PassiFy/Constants.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace PassiFy
```

Comment:
This isn't super necessary since the project is rather tiny but for cleanliness sake I moved the constants out in to their own file so we can access them from anywhere. I also split them by character type.